source/install/install-rhel-8.rst

```diff
@@ -102,4 +102,37 @@ If you wish to remove the Mattermost Server for any reason, you must stop the Ma
 
 	Depending on your configuration, there are several important folders in ``/opt/mattermost`` to backup. These are ``config``, ``logs``, ``plugins``, ``client/plugins``, and ``data``. We strongly recommend you back up these locations before running the ``rm`` command.
 
-You may also remove the Mattermost systemd unit file and the user/group created for running the application.
\ No newline at end of file
+You may also remove the Mattermost systemd unit file and the user/group created for running the application.
+
+Frequently asked questions
+--------------------------
+
+Why doesn't Mattermost start at system boot?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+To have the Mattermost Server start at system boot, the systemd until file needs to be enabled. Run the following command:
+
+.. code-block:: none
+  :class: mm-code-block 
+
+    sudo systemctl enable mattermost.service
+
+Why does Mattermost fail to start at system boot?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+If your database is on the same system as your Mattermost Server, we recommend editing the default ``/lib/systemd/system/mattermost.service`` systemd unit file to add ``After=postgresql.service`` and ``BindsTo=postgresql.service`` to the ``[Unit]`` section.
+
+Can I run Mattermost without a proxy?
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Yes. Mattermost binds to 443 instead of 8065. The Mattermost binary requires the correct permissions to do that binding. You must activate the ``CAP_NET_BIND_SERVICE`` capability to allow the new Mattermost binary to bind to ports lower than 1024 by running the following command:
+
+.. code-block:: none
+  :class: mm-code-block 
+
+    sudo setcap cap_net_bind_service=+ep ./mattermost/bin/mattermost
+
+.. note::
+  
+  - We recommend the `Mattermost Omnibus install method </install/installing-mattermost-omnibus.html>`__ over the tarball if you are running the Mattermost Server and database a single system as this greatly reduces setup and ongoing maintenance.
```

Comment:
@cwarnermm - I feel like because this is RHEL docs, we should remove this line as it's not relevant. @nab-77

Reply:
@cwarnermm yeah. The guidance is about single server but if the admin is on this RHEL page its because they use RHEL :-)